elastic-container.sh

```diff
@@ -87,13 +88,28 @@ get_host_ip() {
 }
 
 set_fleet_values() {
-  fingerprint=$(docker compose exec -w /usr/share/elasticsearch/config/certs/ca elasticsearch cat ca.crt | openssl x509 -noout -fingerprint -sha256 | cut -d "=" -f 2 | tr -d :)
+  fingerprint=$(docker-compose exec -w /usr/share/elasticsearch/config/certs/ca elasticsearch cat ca.crt | openssl x509 -noout -fingerprint -sha256 | cut -d "=" -f 2 | tr -d :)
```

Comment:
This returns an error for me. I think `docker compose exec...` is correct.

Ref: https://github.com/peasead/elastic-container/issues/9

Reply:
Addressed here https://github.com/peasead/elastic-container/pull/12